restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
+h1,h2,h3 {text-align: center;}
+p {text-align: left;}
+</style>
+<body>
+
+<div id="main">
+<h1 id="title"> Becks:
+</h1>
+
+ <p id="def"> 
+    /bɛks/ <i>noun</i> NORTHERN ENGLISH </p>
+     <p id="def"><em>
+        A friendly, curious, passionate (sometimes feisty) , strong, hard-working woman
+     </em></p>
+     <img src="images/IMG_2736.jpg" alt="Becks">
+     
+</div>
+<!--<div>
+<p></p>I love reading
+I love music
+I aspire to be a good mum
+I am a good listener
+I like using my brain -->
+
+
+<div id="music">x
+<h2> Music & Me</h2>
+<p id=def>Music is food for my soul.It turns off my inner chatter and recalibrates me </p>
+<img src="images/kako-0zdex1fYH3MfINRw.png" alt="concert" id=pic>
+<h3> Favourite Piece</h3>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yTyiwtfpO8s" 
+title="YouTube video player" frameborder="0" 
+allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; 
+picture-in-picture" allowfullscreen></iframe>
+<h3> Top Playlist:</h3>
+<iframe allow="autoplay *; encrypted-media *; fullscreen *" frameborder="0" height="450" style="width:100%;max-width:660px;overflow:hidden;background:transparent;" sandbox="allow-forms allow-popups allow-same-origin allow-scripts allow-storage-access-by-user-activation allow-top-navigation-by-user-activation" src="https://embed.music.apple.com/gb/playlist/hiit/pl.u-GgA5e6at89ZgPq"></iframe>
+
+>
+
+<div id="advice"> 
+    <h2>5 Things Motherhood has Taught Me</h2>
+    <ol id="list">
+        <li>Children have an amazing capacity for forgiveness</li>
+        <li>Breathe. And again. Now respond.</li>
+        <li>You teach more through example than any amount of telling</li>
+        <li>There is no limit to the amount of clothes that need to be washed, meals made, stuff tidied....</li>
+        <li> The little stuff matters</li>
+    </ol>
+    </div>
+
+
+<div>
+    <h2>My Favourite Book</h2> 
+    <div class="clearfix">
+
+     
+    <img src="images/The gun seller.jpeg"id="book" > 
+    <p><em>Fantastic read! It's a great page turner. It's very witty but has a great plot too!</em> </p>
```

Comment:
Quote tag would give the effect you are looking for here, reduces from two elements down to one.